packages/mobile/src/transactions/TransferConfirmationCard.tsx

```diff
@@ -119,9 +116,8 @@ function InviteReceivedContent({
         e164PhoneNumber={e164PhoneNumber}
         avatar={
           <ContactCircle
-            name={recipient?.displayName}
```

Comment:
curious why this new syntax didnt work?

Reply:
needed it to return `null` not `undefined`